src/components/tree/tree.common.ts

```diff
@@ -0,0 +1,22 @@
+import IgcTreeItemComponent from './tree-item';
+
+export const IgcTreeSelectionType = {
+  None: 'none',
+  Multiple: 'multiple',
+  Cascade: 'cascade',
+} as const;
+export type IgcTreeSelectionType =
+  typeof IgcTreeSelectionType[keyof typeof IgcTreeSelectionType];
```

Comment:
Do we really need to abstract this?
You already defined it in the tree component as `'none' | 'multiple' | 'cascade'`. Given the items resolve the property from their parent tree this seems unnecessary